tests/test.ts

```diff
@@ -429,3 +429,139 @@ describe('Test clasp logout function', () => {
     expect(dotExists).to.equal(false);
   });
 });
+
+describe('Test clasp help function (in general)', () => {
+  it('should show help correctly', () => {
+    const result = spawnSync(
+      CLASP, ['help'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain('Usage: clasp clasp <command> [options]');
+    expect(result.status).to.equal(1);
+  });
+  it('should show help correctly', () => {
+    const result = spawnSync(
+      CLASP, ['--help'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain('Usage: clasp clasp <command> [options]');
+    expect(result.status).to.equal(0);
+  });
+  it('should show help correctly', () => {
+    const result = spawnSync(
+      CLASP, ['-h'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain('Usage: clasp clasp <command> [options]');
+    expect(result.status).to.equal(0);
+  });
+});
+describe('Test clasp version function', () => {
+  it('should show version correctly', () => {
+    const result = spawnSync(
+      CLASP, ['--version'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain(require('./../package.json').version);
+    expect(result.status).to.equal(0);
+  });
+  it('should show help correctly', () => {
+    const result = spawnSync(
+      CLASP, ['-v'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain(require('./../package.json').version);
+    expect(result.status).to.equal(0);
+  });
+});
+describe('Test clasp unknown function', () => {
+  it('should show version correctly', () => {
+    const result = spawnSync(
+      CLASP, ['unknown'], { encoding: 'utf8' },
+    );
+    expect(result.stderr).to.contain('🤔  Unknown command "unknown"');
+    expect(result.status).to.equal(1);
+  });
+});
+
+describe('Test all functions while logged out', () => {
+  it('should list error correctly', () => {
+    const result = spawnSync(
+      CLASP, ['list'], { encoding: 'utf8' },
+    );
+    expect(result.status).to.equal(1);
+    expect(result.stderr).to.contain('Could not read API credentials. Error:');
```

Comment:
This error comes from here:

https://github.com/google/clasp/blob/87b8de5ed6b31dfbbf145ba121b41ec18404d45d/src/auth.ts#L91

Why not put it in this?

https://github.com/google/clasp/blob/87b8de5ed6b31dfbbf145ba121b41ec18404d45d/src/utils.ts#L82

Then we can import it as `ERROR.NO_CREDENTIALS` or something like that.

Reply:
Yes, please!